src/libraries/Microsoft.Extensions.DependencyInjection/tests/Utils/MultiServiceHelpers.cs

```diff
@@ -43,7 +42,7 @@ private static bool IsGenericIEnumerable(Type type)
 
         private static Type FirstGenericArgument(Type type)
         {
-            return type.GetTypeInfo().GenericTypeArguments.Single();
+            return type.GetTypeInfo().GenericTypeArguments[0];
```

Comment:
Isn't this a behavioral change? Seems like it's the equivalent of `First()`, not `Single()`.

Reply:
Ah, this code path is only used for `IEnumerable<>` cases. Carry on. :)